scripts/base/frameworks/storage/async.zeek

```diff
@@ -25,7 +25,7 @@ export {
 	## Returns: A record containing the status of the operation, and either an error
 	##          string on failure or a value on success. The value returned here will
 	##          be an ``opaque of BackendHandle``.
-	global open_backend: function(btype: Storage::Backend,
+	global open_backend: function(btype: Storage::Backend, stype: Storage::Serializer,
```

Comment:
WDYT about putting the serializer into `BackendOptions`? 
```
type BackendOptions: record {
    serializer: Storage::Serializer &default=Storage::SERIALIZER_JSON
};
```
Or possibly event `&default=default_serializers` where the latter is `const &redef` within `Storage`.

Seems reasonable to me to have a default that works well out of the box, but optionally allow injecting something else.

Reply:
Yeah, I like that more actually, as it removes the need to pass the serializer along through the constructors too.